src/lpython/pickle.cpp

```diff
@@ -148,6 +175,52 @@ class ASRTreeVisitor :
         return s;
     }
 
+    void visit_TranslationUnit(const ASR::TranslationUnit_t &x) {
+        if(!attached) {
+            attached = true;
+        }
```

Comment:
@ansharlubis What is the use of the attached used for?
Also, please go through these changes, if you see anything that needs to be changed or reverted, let me know.

Reply:
What is `attached` for?

Reply:
I think in a previous iteration of the pretty printer, functions were printed as `Function=function_name`.
I don't remember how I implemented it, but I think that's what `attached` was for.

Reply:
Sorry, ignore my previous comment. `attached` is used to differentiate printing
```
Name
|- ....
```
from
```
|-field=Name
  |- ...
```